stories/SQForm.stories.js

```diff
@@ -197,6 +211,13 @@ export const BasicForm = () => {
         >
           {CHECKBOX_GROUP_OPTIONS}
         </SQFormCheckboxGroup>
+        <SQFormMultiValue
+          name="favoriteColors"
+          label="Your Favorite Colors"
+          size={12}
```

Comment:
Size 12 is a little extreme.  I don't think we will be doing that large of a field in any of our apps so we probably shouldn't do that in our Storybook as well.  We want to keep the stories as close to real world examples as possible.

Reply:
+1
I've never realized how wide that form gets if the right side bar in Storybook is hidden.
Branden all up in here with the ultra wide lol.

Can we make this a 4 or something? To try and get a realistic view of how it'll behave in our apps.
Or maybe we add a max-width to the form wrapper so it doesn't grow so wide.

Reply:
![Screen Shot 2021-06-17 at 8 57 34 AM](https://user-images.githubusercontent.com/29528409/122411065-13b39d80-cf4a-11eb-86f2-7aa5b69d911d.png)